spec/directives/if-else.hrx

```diff
@@ -0,0 +1,35 @@
+<===> capitalized_else/input.scss
+@if false {}
+@Else {a {b: c}}
+
+<===> capitalized_else/output.scss
+@Else {
+  a {
+    b: c;
+  }
+}
+
+<===> escaped/options.yml
```

Comment:
Style nit: [use comment dividers to separate specs](https://github.com/sass/sass-spec/blob/master/STYLE_GUIDE.md#do-use-comment-dividers-to-separate-specs-within-an-hrx-file).